### ISSUE TITLE
Upgrade from 2.5.1 to 2.9.0 causes spelling errors for "iframe"

### ISSUE THREAD

**Issue by faunaee** [vale-cli/vale#325]
- macOS 11 (Big Sur)
- Install method: goreleaser
- Vale version:
  ```
  $ bin/vale -v
  vale version 2.9.0
  ```

With Vale 2.5.1, I used this style to perform spell checking:
```yaml
extends: spelling
message: "'%s'?"
aff: ../../dictionaries/en_US-large.aff
dic: ../../dictionaries/en_US-large.dic
level: error
ignore:
  - ../dictionaries/custom_words.txt
```

With Vale 2.9.0, I updated the style to use the new keys for the `spelling` definition (I hope to take advantage of multiple dictionaries):
```yaml
extends: spelling
message: "'%s'?"
dicpath: ci/dictionaries
dictionaries:
  - en_US-large
level: error
ignore:
  - ci/dictionaries/custom_words.txt
```

With Vale 2.5.1, my result from Vale processing is:
```
✔ 0 errors, 0 warnings and 0 suggestions in 3949 files.
```

With Vale 2.9.0, my result from Vale processing is:
```
✖ 692 errors, 0 warnings and 0 suggestions in 3949 files.
```

The vast majority of the errors are:
```
 build/some_page.html
 56:12  error  'iframe'?  MyOrg.Spelling 
 57:65  error  'iframe'?  MyOrg.Spelling 
```

Those lines correspond with this HTML:
```html
56: <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-*******"
57:   height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

If I use the 2.5.1 style for spelling with Vale 2.9.0, my result from Vale processing is:
```
✔ 0 errors, 0 warnings and 0 suggestions in 3949 files.
```

If I use the 2.9.0 style for spelling, but use invalid paths for the dictionary, I get this error:
```
E201 Invalid value provided [ci/vale/MyOrg/Spelling.yml:1:1]:

   1* extends: spelling
   2  message: "'%s'?"
   3  dicpath: ci/dictionary

open /Users/me/repos/fixes/ci/dictionary/en_US-large.dic: no such file or directory
```

So, it would seem that Vale 2.9.0 is finding/using the specified dictionary when using the new keys for the `spelling` style, but there's some additional change that results from using the new keys themselves.

You might suggest "just use the old `spelling` keys", but that would prevent me from using multiple dictionaries, which is the reason that I would upgrade to 2.9.0.

**Comment by jdkato** [comment]
In v2.5.1, did you have to explicitly ignore `iframe` or did it just not raise an error? 

It seems like the problem here is that `iframe` shouldn't be raising an error, regardless of what dictionary you're using.

**Comment by faunaee** [comment]
For 2.5.1, no error was raised. "iframe" does not appear in the dictionary nor in the `custom_words.txt` file.

**Comment by jdkato** [comment]
Should be fixed in v2.9.1.

**Comment by faunaee** [comment]
Awesome! I've just verified that the fix in 2.9.1 does, indeed, solve this problem. I'm now using multiple dictionaries successfully.

Thanks for fixing this so quickly!
